clusterloader2/pkg/measurement/util/gatherers/container_resource_gatherer.go

```diff
@@ -27,8 +27,8 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	clientset "k8s.io/client-go/kubernetes"
 	"k8s.io/klog"
-	"k8s.io/kubernetes/pkg/util/system"
 	"k8s.io/perf-tests/clusterloader2/pkg/measurement/util"
+	cl2 "k8s.io/perf-tests/clusterloader2/pkg/util"
```

Comment:
cl2 is misleading - the previous is also cl2.
Can you please name alias it by "pkgutil"? (we generally use convention to use suffix of path as an alias).

BTW - it might be useful to move this part to a separate PR, since it's unrelated and the risk of breaking is very low.

Reply:
@wojtek-t it has to be coupled with this PR otherwise the build fails because legacyIsMasterNode is no longer available publicly